class/Zarieth; The Witch

```diff
@@ -0,0 +1,1836 @@
+									]
+								}
+							]
+						}
+					],
+					[
+						{
+							"entries": [
+								{
+									"name": "Forceful Presence",
+									"entries": [
+										"When you reach 6th level, you can retain control over a spirit after it produces its release effect so that it remains bound and ready to be released again.",
+										"You may use this ability a number of times equal to your Charisma modifier (minimum of once). You recover all expended uses after a long rest."
+									]
+								}
+							]
+						}
+					],
+					[
+						{
+							"entries": 
+							[
+								{
+									"name": "Powerful Bonds",
+									"entries": 
+									[
+										"At 10th level, your body and soul develop the ability to contain more powerful spirits. The level of spirits you bind are considered 1 higher than the spell slot you use to bind them."
+									]
+								}
+							]
+						}
+					],
+					[
+						{
+							"entries": 
+							[
+								{
+								"name": "Pale Host",
+								"entries": 
+									[
+										"At 14th level, when you are reduced to 0 hit points but not killed outright, your body becomes possessed by a spirit you control and you may continue to act for 1 round per level of the highest level spirit you have in your power. You cannot use this ability if you do not have any spirits bound. During this time, your hit points remain at 0 and you continue to make death saving throws (though a prepared witch may move to her spirit ward to become stable).",
+										"Onlookers notice a strangeness in your movements, eyes, and voice while you remain possessed. While possessed, you can use your action on your turn to activate the release power of one of the spirits you have bound and have it remain bound. This does not expend any uses of the Forceful Presence ability.",
+										"Should you be healed under the possession (including by yourself), you awaken from the possession, conscious. Otherwise, when the duration expires, your body collapses."
+									]
+								}
+							]
+						}
+					]
+				]
+			}
+		]
+	}
+
+],
+"optionalfeature": 
+[
+	{
+		"type": "optfeature",
+		"name": "Agony",
+		"source": "The Witch",
+		"page": 10,
+		"featureType": "Spirit Binding",
+		"entries": [
+			"In the presence of a spirit of Agony, foes’ wounds continue to bleed and fires continue to burn. While Agony is bound, for every damage die you roll that shows a 1, you may treat the roll as the maximum possible number for that die for a number of dice up to the level of the spirit.",
+			"{@b {@i Release.}} You may release Agony as part of any attack or spell which deals damage. In addition to the damage, you may roll an extra number of equivalent dice equal to the level of the spirit. If the attack or spell uses more than one type of die, choose the largest. This extra damage is psychic damage. Additionally, all 1s are treated as the maximum value for each die."
```

Comment:
This is better represented as:
```json
{
  "type": "entries",
  "name": "Release",
  "entries": [
    "You may release Agony as part of any attack or spell which deals damage. In addition to the damage, you may roll an extra number of equivalent dice equal to the level of the spirit. If the attack or spell uses more than one type of die, choose the largest. This extra damage is psychic damage. Additionally, all 1s are treated as the maximum value for each die."
  ]
}
```
(and the same for the others)

Reply:
Changed for all the occurences. Wasn't sure how i could properly get the stylemark like this. Now I'll know for future conversions.